### PR TITLE
SDG-1: Add Elasticsearch & Kibana 7.

### PR DESCRIPTION
**Pull request by tormi**
This PR adds Elasticsearch & Kibana 7 setup to the project template. ES is using [Lando's internal ES service](https://docs.devwithlando.io/tutorials/elasticsearch.html), Kibana is using [blacktop/kibana](https://hub.docker.com/r/blacktop/kibana/) image.

Screenshot 2019-07-19 at 17 48 00
Screenshot 2019-07-19 at 17 49 22
Screenshot 2019-07-19 at 17 48 55

### CODE REVIEW

File: .lando/elasticsearch.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+http:
```
Comment (floretan) [inline .lando/elasticsearch.yml]: Why do we need to specify these values? What part varies from the defaults?

Comment (tormi) [reply .lando/elasticsearch.yml]: I've used [Lando's example config](https://github.com/lando/lando/blob/master/examples/elasticsearch/config/custom.yml) and added `cors` settings in addition to that. What's your suggestion, remove `elasticsearch.yml` altogether?

Comment (tormi) [reply .lando/elasticsearch.yml]: I'll review the defaults and remove those.

File: .lando/elasticsearch.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  name: cluster_name
+node:
+  name: node_name
+  master: true
+  data: true
+  ingest: true
+discovery:
+  type: single-node
+plugin:
+  mandatory: analysis-icu
```
Comment (tormi) [inline .lando/elasticsearch.yml]: This duplicate generates following error:

```
elasticsearch_1    |  11:01:37.41 INFO  ==> ** Starting Elasticsearch setup **
elasticsearch_1    |  11:01:37.43 DEBUG ==> Validating Kernel settings...
elasticsearch_1    |  11:01:37.43 DEBUG ==> Validating settings in ELASTICSEARCH_* env vars...
elasticsearch_1    |  11:01:37.44 INFO  ==> Configuring/Initializing Elasticsearch...
elasticsearch_1    |  11:01:37.45 DEBUG ==> Detected persisted data from previous deployments
elasticsearch_1    |  11:01:37.45 DEBUG ==> Ensuring expected directories/files exist...
elasticsearch_1    |  11:01:37.47 INFO  ==> Custom configuration file detected, using it...
elasticsearch_1    |  11:01:37.47 DEBUG ==> Using specified values for Xmx and Xms heap options...
elasticsearch_1    |  11:01:37.48 DEBUG ==> Setting '-Xmx1026m -Xms1026m' heap options...
elasticsearch_1    |  11:01:37.49 DEBUG ==> Installing plugins: analysis-icu
elasticsearch_1    |  11:01:37.58 DEBUG ==> Installing plugin: analysis-icu
elasticsearch_1    | Checking if url exists: https://artifacts.elastic.co/downloads/elasticsearch-plugins/analysis-icu/analysis-icu-linux-x86_64-7.2.0.zip
elasticsearch_1    | -> Downloading analysis-icu from elastic
elasticsearch_1    | Retrieving zip from https://artifacts.elastic.co/downloads/elasticsearch-plugins/analysis-icu/analysis-icu-7.2.0.zip
elasticsearch_1    | WARNING: An illegal reflective access operation has occurred
elasticsearch_1    | WARNING: Illegal reflective access by org.bouncycastle.jcajce.provider.drbg.DRBG (file:/opt/bitnami/elasticsearch/lib/tools/plugin-cli/bcprov-jdk15on-1.61.jar) to constructor sun.security.provider.Sun()
elasticsearch_1    | WARNING: Please consider reporting this to the maintainers of org.bouncycastle.jcajce.provider.drbg.DRBG
elasticsearch_1    | WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
elasticsearch_1    | WARNING: All illegal access operations will be denied in a future release
elasticsearch_1    | ERROR: plugin directory [/opt/bitnami/elasticsearch/plugins/analysis-icu] already exists; if you need to update the plugin, uninstall it first using command 'remove analysis-icu'
```

Comment (tormi) [reply .lando/elasticsearch.yml]: d672fc0 resolves the issue above.

File: .lando/elasticsearch.yml (hunk 1 above)
Comment (tormi) [reply .lando/elasticsearch.yml]: Ok, I'll remove `elasticsearch.yml` and refer to docs. These are default values:

```
lando ssh -s elasticsearch -u root -c "cat /opt/bitnami/elasticsearch/config/elasticsearch.yml"
http:
  port: 9200
path:
  data: /bitnami/elasticsearch/data
transport:
  tcp:
    port: 9300
network:
  host: |-
    172.20.0.3
    172.28.0.6
  publish_host: |-
    172.20.0.3
    172.28.0.6
  bind_host: 0.0.0.0
cluster:
  name: bespin
node:
  name: lando
  master: true
  data: true
  ingest: false
discovery:
```

Comment (tormi) [reply .lando/elasticsearch.yml]: @floretan , resolved here: b809fe3.

File: .lando.yml [hunk 3]
```diff
@@ -53,3 +73,6 @@ events:
   # Clear caches after a database import
   post-db-import:
     - appserver: cd $LANDO_WEBROOT && drush cr -y
+
+# Lando version at the time of creation of .lando.yml.
+version: v3.0.0-rc.18
```
Comment (floretan) [inline .lando.yml]: Very nice, documenting this is a great idea!

**Pull request merged**